### PR TITLE
(blog) Bulding multilingual sites with Gatsby

### PR DESCRIPTION
**Pull request by ondrabus**
Blog post topic pre-approved by Hashim Warren

### CODE REVIEW

File: docs/blog/2020-02-19-how-to-build-multilingual-sites-with-gatsby/index.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+║ /cs/home                                           ║
+║ /sk/home                                           ║
+╚════════════════════════════════════════════════════╝
+```
+
+I chose the variant with default language having no prefix in the URL.
+
+## Identification of User Locale
+
+If your website is not Google Maps, people most likely won't share their location with you. Therefore, when identifying where geographically a visitor is, we rely on the information browsers send in the first request. That can be obtained via `window.navigator.language`, but we typically don't implement this part ourselves. In my case, it's a job that is handled by the mentioned `gatsby-plugin-i18n`.
```
Comment (AishaBlake) [inline docs/blog/2020-02-19-how-to-build-multilingual-sites-with-gatsby/index.md]:
```suggestion
If your website is not Google Maps, people most likely won't share their location with you. Therefore, when identifying where a visitor is geographically, we rely on the information browsers send in the first request. That can be obtained via `window.navigator.language`, but we typically don't implement this part ourselves. In my case, it's a job that is handled by the aforementioned `gatsby-plugin-i18n`.
```

File: docs/blog/2020-02-19-how-to-build-multilingual-sites-with-gatsby/index.md [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+## Static Language-specific Pages
+
+Apart from dynamic pages, there are always some static pages. They include `index.js` and `employees.js` that handle the homepage and employees page respectively. The `gatsby-plugin-i18n` will place them on the right language-specific URLs if you follow the defined language convention-the filename suffix needs to contain the language code.
+
+* index.js -> index.en.js, index.cs.js
+* employees.js -> employees.en.js, employees.cs.js
+
+It's also necessary to adjust the content of each of the new files to reflect its new language. That includes component properties. Take a look at this part of my index.js:
+
+```xml
```
Comment (AishaBlake) [inline docs/blog/2020-02-19-how-to-build-multilingual-sites-with-gatsby/index.md]:
```suggestion
```js:title=index.js
```

File: docs/blog/2020-02-19-how-to-build-multilingual-sites-with-gatsby/index.md [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+<Layout location={location} title={title}>
+     <IndexContent />
+ </Layout>
+```
+
+Once this file becomes `index.cs.js`, I need to adjust it to:
+
+```xml
```
Comment (AishaBlake) [inline docs/blog/2020-02-19-how-to-build-multilingual-sites-with-gatsby/index.md]:
```suggestion
```js:title=index.cs.js
```

File: docs/blog/2020-02-19-how-to-build-multilingual-sites-with-gatsby/index.md [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+         langKeyForNull: 'en',
+         prefixDefault: false,
+         useLangKeyLayout: false,
+     },
+ },
+```
+
+This configuration (in `gatsby-config.js`) tells the plugin to use 'en' as the default language code (`langKeyDefault`, `langKeyForNull`) and no prefix (`prefixDefault`). The last option (`useLangKeyLayout`) specifies that the used layout is language invariant.
+
+## Localized URLs
```
Comment (AishaBlake) [inline docs/blog/2020-02-19-how-to-build-multilingual-sites-with-gatsby/index.md]: Seems like this and the following four headers are all one section and should be `h3`s. What do you think?

Comment (ondrabus) [reply docs/blog/2020-02-19-how-to-build-multilingual-sites-with-gatsby/index.md]: Hi Aisha, wow, thank you so much for the suggested changes! Yes, I will adjust the headers :)

**Comment by ondrabus** [comment]
Thank you @AishaBlake, all the suggestions should now be resolved.

**Comment by ondrabus** [comment]
@AishaBlake @hashimwarren is there anything else I should adjust before merging please?

**Pull request merged**